extensions/wikia/Recirculation/templates/client/rail.mustache

```diff
@@ -2,7 +2,7 @@
 	<h2>{{{ titleHtml }}}</h2>
 	<ul class="thumbnails">
 		{{#items}}
-			<li class="rail-item {{classes}}" data-flag="{{ flag }}" data-index="{{ index }}" data-source="{{source}}">
+			<li class="rail-item {{classes}} item-{{source}}" {{#id}}data-id="{{ id }}"{{/id}} data-flag="{{ flag }}" data-index="{{ index }}" data-source="{{source}}">
```

Comment:
I'm not sure what are the guidelines for the mustache files, but I'd probably add spaces inside `{{#id}}...{{/id}}`